README.md

```diff
@@ -17,27 +17,49 @@ Dash is a powerful new peer-to-peer platform for the next generation of financia
 npm install dashcore-lib
 ```
 
-## Documentation
+## Docs
+
+* [Addresses](address.md)
+* [Block](block.md)
+* [Crypto](crypto.md)
+* [Encoding](encoding.md)
+* [Hierarchically-derived Private and Public Keys](hierarchical.md)
+* [Networks](docs/networks.md)
+* [PrivateKey](docs/privatekey.md)
+* [PublicKey](docs/publickey.md)
+* [Script](docs/script.md)
+* [Transaction](docs/transaction.md)
+* [Using Different Units](unit.md)
+* [Unspent Output](docs/upspentoutput.md)
+* [URI](docs/uri.md)
+* [Governance Object / Proposal](govobject/govobject.md)
 
-The complete docs are hosted here: [bitcore documentation](http://bitcore.io/guide/). There's also a [bitcore API reference](http://bitcore.io/api/) available generated from the JSDocs of the project, where you'll find low-level details on each bitcore utility.
+## Examples
 
-- [Read the Developer Guide](http://bitcore.io/guide/)
-- [Read the API Reference](http://bitcore.io/api/)
+Some exemple can be find [here](docs/examples.md), below is a list of direct link for some of them.
```

Comment:
One more "example" typo.
Also, minor grammatical change. Suggest changing:
> Some exemple can be find [here](docs/examples.md), below is a list of direct link for some of them.

to

> Some examples can be found [here](docs/examples.md), below is a list of direct links for some of them.

Reply:
Nice ! Fixed !